cmd/nada-backend/main.go

```diff
@@ -110,6 +110,8 @@ func main() {
 
 	mockHTTP := api.NewMockHTTP(repo, log.WithField("subsystem", "mockhttp"))
 	var httpAPI api.HTTPAPI = mockHTTP
+	api.Init(repo.Querier)
```

Comment:
This has nothing to do with the PR, but my guess is that it is just to simplify the implementation, the actual logic might be:
if cfg.MockAuth {
    httpAPI = mocked http
}
else
{
    httpAPI = real http
}

but go requires the variable to be defined first, so there will be an extra line to define httpAPI, then why dont we assign a value when defining it, so you saw what you saw.

just my guess, because if I were the author I would take such a reason:)